playbooks/openshift-glusterfs/private/setup_nodes.yml

```diff
@@ -10,6 +10,7 @@
   - import_role:
       name: openshift_node
       tasks_from: glusterfs.yml
+  - import_tasks: roles/openshift_node/tasks/storage_plugins/iscsi.yml
```

Comment:
```suggestion
  - import_role: openshift_node
    tasks_from: storage_plugins/iscsi.yml
```

Reply:
I tried this first, but travis complained. Some linter couldn't find `file 'iscsi.yml'`.

Reply:
Ah, okay, lets stick to this then

Reply:
The problem is that `tasks_from:` cannot reference files in subdirectories under `tasks\`.  There is an Ansible bug on this somewhere.